packages/cli/test/integration-2.test.ts

```diff
@@ -1187,17 +1187,15 @@ test('[vc build] should build project with `@vercel/speed-insights`', async () =
   }
 });
 
-test('[vc build] should not build project with `@vercel/analytics globally installed`', async () => {
+test('[vc build] should build project with `@vercel/analytics globally installed`', async () => {
```

Comment:
Minor nit, but I don't think the phrasing "globally" is correct here. To me, globally means with the `-g` flag (`npm i -g ...`), whereas here you mean higher up in the workspace hierarchy. I'd maybe say "at the root level" or "in a parent workspace"